test/plots/piePlotTests.ts

```diff
@@ -143,6 +143,46 @@ describe("Plots", () => {
       });
     });
 
+    describe("Rendering with scale", () => {
```

Comment:
How come this is a separate `describe()` instead of just an `it()` under an existing `describe()`?

It should be possible to override the old `sectorValue()` call with the new one.